tests/wpt/metadata/html/semantics/links/links-created-by-a-and-area-elements/htmlanchorelement_noopener.html.ini

```diff
@@ -2,5 +2,14 @@
   type: testharness
   expected: ERROR
   [Check that rel=noopener with target=_self does a normal load]
+    expected: FAIL
+
+  [Check that targeting of rel=noopener with a given name ignores an existing window with that name]
     expected: NOTRUN
 
+  [Check that rel=noopener with target=_parent does a normal load]
+    expected: FAIL
+
+  [Check that rel=noopener with target=_top does a normal load]
+    expected: FAIL
+
```

Comment:
This is a loop creating three tests, which uses `window.open`, and previously would fail/error in the first iteration.